atom/browser/native_window.h

```diff
@@ -144,6 +148,10 @@ class NativeWindow : public content::WebContentsObserver,
                               const std::string& description) = 0;
   virtual void SetVisibleOnAllWorkspaces(bool visible) = 0;
   virtual bool IsVisibleOnAllWorkspaces() = 0;
+#if defined(OS_WIN)
```

Comment:
My original thought: this API requires the `ThumbarHost::ThumbarButton` structure which should be hidden frm other two platforms.
So do you think moving the `ThumbarButton` to `NativeWindow`  is acceptable?